plugins/woocommerce/src/Internal/Admin/Orders/ListTable.php

```diff
@@ -565,6 +567,48 @@ private function set_search_args(): void {
 		}
 	}
 
+	/**
+	 * Implements filtering of orders by created_via value.
+	 */
+	private function set_created_via_args(): void {
+		// phpcs:disable WordPress.Security.NonceVerification.Recommended
+		$created_via = sanitize_text_field( wp_unslash( $_GET['_created_via'] ?? '' ) );
+
+		if ( empty( $created_via ) ) {
+			return;
+		}
+
+		$this->order_query_args['created_via'] = explode( ',', $created_via );
+		$this->has_filter                      = true;
+	}
+
+	/**
+	 * Render the created_via filter dropdown.
+	 *
+	 * @return void
+	 */
+	public function created_via_filter() {
+		// phpcs:disable WordPress.Security.NonceVerification.Recommended
+		$current_created_via = isset( $_GET['_created_via'] ) ? sanitize_text_field( wp_unslash( $_GET['_created_via'] ) ) : '';
+
+		$created_via_options = array(
```

Comment:
I was also wondering about a filter here, but I'm good holding off on this until it's requested.

If we do create a POS class at some point, it might make sense to move these options into a getter in that class, but we can old off on this if that class/feature is not set up yet.

Reply:
Yeah, I asked about that in a previous [comment](https://github.com/woocommerce/woocommerce/pull/57273#discussion_r2044590480). But I think you are right, we can hold off on this a bit 👍